datasets/sondehub-telemetry.yaml

```diff
@@ -0,0 +1,55 @@
+Name: SondeHub Radiosonde Telemetry
+Description: >
+  SondeHub Radiosonde telemetry contains global radiosonde (weather balloon) data captured by SondeHub from our
+  participating radiosonde_auto_rx receiving stations. Currently 313 receiver stations are providing
+  data for an average of 384 radiosondes a day.
+
+  Data contains telemetry frames in Universal Sonde Telemetry format, including gps position, type,
+  altitude, pressure, temperature and humidity.
+Documentation: https://github.com/projecthorus/sondehub-infra/wiki/Amazon-Open-Data
+Contact: Michaela Wheeler <vk3fur@sondehub.org>
+ManagedBy: SondeHub
+UpdateFrequency: Data is updated as we receive it
+Tags:
+  - climate
+  - environmental
+  - weather
+  - GPS
+License: Creative Commons BY-SA 2.0
+Resources:
+  - Description: Radiosonde Telemetry as JSON blobs of Universal Telemetry format prefixed by date/time
+    ARN: arn:aws:s3:::sondehub-open-data
+    Region: us-east-1
+    Type: S3 Bucket
+    Explore: http://sondehub-open-data.s3-website-us-east-1.amazonaws.com/#serial/
+  - Description: Radiosonde Telemetry as JSON blobs of Universal Telemetry format prefixed by serial number
```

Comment:
Lets just list the bucket once and then Explore should be an array of links so you can have By Date/time & Serial Number options there